climatemind.py

```diff
@@ -1,16 +1,17 @@
 from knowledge_graph import app, db
-from knowledge_graph.models import User, Scores, Lrf, Zip, Iri
+from knowledge_graph.models import Users, Scores, Sessions  # TODO Lrf, Zip, Iri
 
 
 @app.shell_context_processor
 def make_shell_context():
     return {
         "db": db,
-        "User": User,
+        "User": Users,
         "Scores": Scores,
-        "LRF": Lrf,
-        "Zip": Zip,
-        "Iri": Iri,
+        "Sessions": Sessions,
```

Comment:
Is there supposed to be a comma after Sessions?

Reply:
commas after list are okay if they are on their own line. Very handy trick to remember. Makes it easier to add items/comment out other items in the future.